content/05.heterogeneity.md

```diff
@@ -1,3 +1,20 @@
 ### Techniques to manage disparities in data generation are required to power robust analyses in rare diseases
->Rarity of patients leads to heterogeneity in sample collection, causing disparities in the data. 
->We will discuss how rigorous normalization and methodologies capturing sample-wise gene-set level information can help appropriate integration of disparate data points to power machine learning approaches11–13. 
+
+As with common diseases, genomic and transcriptomic data from rare diseases can suffer from artifacts introduced by batch, processing methodology, sequencing platform, or other non-biological phenomena. 
+This effect is amplified in rare diseases which often have few samples and heterogenous phenotypes.
+Furthermore, because datasets are many times pieced together from multiple small studies, disease phenotype or other important biological characteristics are often confounded by the previously mentioned "batch" factors. 
+A key consideration here is, if possible, active dialogue with the data generators or experts in the field who may have unexpected insights into potential sources of variation. 
+One example of the value of this, experienced by the authors, occurred when studying tumors associated with the disease neurofibromatosis type 1. 
+These datasets were, unbeknownst to the computational biologists, generated from samples obtained with vastly different surgical techniques (laser ablation and excision vs standard excision), resulting in substantial biological differences that are a consequence of process, not reality. 
```

Comment:
This is a really good point. Can this phenomenon be included in batch effects? If not, is there a good data/metadata check that we can think of for new practitioners? you have already noted methodology review with collaborators/data-contributors, adding: methods section review for published data.

Reply:
I think this is technically a "batch effect," but it's also more than just a technical artifact since there is true biology. Presumably, in this example, it would cause the activation of heat shock protein related pathways, unfolded protein responses, etc. The same could be said for prolonged warm ischemia time, etc. 

I'm not clear what you mean by this: 
>If not, is there a good data/metadata check that we can think of for new practitioners?

I think the key lesson here for readers is that understanding the normalization challenges isn't always as simple as having comprehensive sample metadata (i.e. it will never be comprehensive enough -there's always a corner case). There may be more intangible pieces of information that an expert - in this case, a clinician - can provide to bring context to the problem.

Reply:
I like your example here:
> I think this is technically a "batch effect," but it's also more than just a technical artifact since there is true biology. Presumably, in this example, it would cause the activation of heat shock protein related pathways, unfolded protein responses, etc. The same could be said for prolonged warm ischemia time, etc.

It may be helpful to include this perspective in the text to help readers understand the biological implication of this instead of thinking of it as just another batch effect/technical artifact.

Reply:
Great idea! done. I didn't want to belabor this point too much so I just added a sentence. I don't think it's a critical distinction to the reader.